src/util/mpi_utils.c

```diff
@@ -155,6 +156,7 @@ void mpi_one_fprintf(FILE *f, const char *template, ...)
 	  va_start(ap, template);
 	  vfprintf(f, template, ap);
 	  va_end(ap);
+       fflush(f);
```

Comment:
fix indent

Reply:
Done; the previous lines were partly indented with tabs; converted to spaces as elsewhere in the file.